content/blog/kubernetes-chart-v4/index.md

```diff
@@ -0,0 +1,1033 @@
+---
+title: "New: Helm Chart v4 resource with tons of new features"
```

Comment:
"tons of new features" feels a little weak, but I am not sure what would be better. Maybe "Install Helm charts with precision using the new Helm Chart v4 resource"? Not sure if "precision" is quite right, just thinking that we should emphasize the main benefits of using the Chart resource in the title.

Reply:
Agree, I also might suggest you swap the order of your two opening paragraphs and summarize a few of the features so that someone skimming the intro knows what's being delivered. i.e. :  ... brings a host of new features, including enhanced SDK support across all Pulumi SDKs, full OCI registry support, improved handling of chart values, better connectivity for cluster interactions, improved resource ordering, and support for new-style Pulumi transformations.